src/Symfony/Component/Messenger/Bridge/Kafka/composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "symfony/kafka-messenger",
+    "type": "symfony-bridge",
+    "description": "Symfony Messenger Kafka Transport Bridge",
+    "keywords": ["kafka", "symfony", "messenger", "transport", "queue", "bridge"],
+    "license": "MIT",
+    "require": {
+        "php": ">=7.2.5",
+        "ext-rdkafka": "^4.0",
+        "symfony/messenger": "^5.0",
```

Comment:
Hello, just a little question : why only for symfony/messenger 5 and not 4 and 5 ?